themes/default/content/learn/glossary/gcp-gke.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Google Kubernetes Engine
+meta_desc: A managed container orchestration tool on Google Cloud Platform (GCP), allowing for the deployment and automated management of containerized applications.
+layout: glossary/single
+---
+
+## Description
+
+[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) (GKE) is a managed container orchestration tool. Applications are deployed within containers by defining a cluster and node pool, then giving is a kubeconfig, deployment definition, and service definition, as with any other application deployed on a Kubernetes cluster. GKE also provides some additional services, such as auto-repair for unhealthy nodes and and auto-scaling. Serverless functionality is available via [Cloud Run](https://cloud.google.com/run) for Anthos.
```

Comment:
I'm having trouble parsing the the second sentence. Should this is `it`? (And if so, does "it" refer to the cluster or the node pool?)

```suggestion
[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) (GKE) is a managed container orchestration tool. Applications are deployed within containers by defining a cluster and node pool, then giving the cluster a kubeconfig, deployment definition, and service definition, as with any other application deployed on a Kubernetes cluster. GKE also provides some additional services, such as auto-repair for unhealthy nodes and auto-scaling. Serverless functionality is available via [Cloud Run](https://cloud.google.com/run) for Anthos.
```